packages/core/src/App/Components/Layout/Header/account-actions.jsx

```diff
@@ -38,19 +43,27 @@ const AccountActions = React.memo(
         toggleAccountsDialog,
         toggleNotifications,
         is_deposit_button_disabled,
+        is_mobile,
     }) => {
         const { is_appstore } = React.useContext(PlatformContext);
 
-        if (is_logged_in) {
-            return (
+        const { is_wallet_enabled } = useFeatureFlags();
+        const { has_wallet, isLoading } = useWalletAccountsList();
+
+        const should_show_wallets = is_wallet_enabled && has_wallet;
+
+        if (is_logged_in && !isLoading) {
+            return is_mobile ? (
                 <React.Fragment>
-                    <MobileWrapper>
```

Comment:
@arshad-rao-deriv @sergei-deriv why did you remove these Desktop and Responsive wrappers?

Reply:
@mahdiyeh-deriv it's because I'm using is_mobile from the store instead of the wrappers